### PR TITLE
Feat/update deps

### LINKED ISSUE #49: [Task]: Upgrade dependencies

### PR DESCRIPTION
**Pull request by aspnxdd**
Fixes #49

## Changes

  - Update deps
  - Migrate from yarn to pnpm
  - Fix 2 lint errors to compile project without errors.
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,35 +13,37 @@
...
-    "next": "12.1.6",
-    "react": "18.1.0",
-    "react-dom": "18.1.0",
-    "sass": "^1.54.0",
+    "mongoose": "^6.8.4",
+    "next": "13.1.5",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "sass": "^1.57.1",
     "styled-components": "^5.3.5"
```
Comment (evavirseda) [inline package.json:35]: could this be updated? 
image

**Review by evavirseda — CHANGES_REQUESTED** [review]
Looks great! but i have a question, why are we migrating to pnpm?

**Comment by aspnxdd** [comment]
> Looks great! but i have a question, why are we migrating to pnpm?

I personally prefer it :smile_cat:  and it is quite faster than yarn!

**Review by evavirseda — APPROVED** [review]
LGTM!

**Pull request merged**